requirements.txt

```diff
@@ -5,3 +5,4 @@ pyserial
 future
 requests
 pywin32; platform_system=="Windows"
+suds
```

Comment:
Note I added suds here so we get it installed in the build and on people's machines when they do this from source.

Reply:
Thanks, right!